modules/ROOT/pages/manage-blocked-ip-in-rtf.adoc

```diff
@@ -0,0 +1,81 @@
+= Manage Blocked IP Addresses in Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In the Runtime Fabric *Inbound Traffic > Security Policies* interface, you can unblock IP addresses and generate a list of blocked IP addresses, which you can view in Runtime Fabric logs. 
+
+[NOTE]
+You must have a DoS policy configured to manage IP addresses in Runtime Fabric. See xref:dos-policy.adoc[Denial of Service Policy]
+
+. In Anypoint Platform go to *Management Center > Runtime Manager*.
+. In the left navigation menu, click *Runtime Fabrics*.  
+. Select the Runtime Fabric for which to manage the blocked IP addresses. 
+. Select the *Inbound Traffic* tab. 
+. In the *Security Policies* section, click *Manage blocked IPs*.
+. In the *Manage blocked IPs* dialog box, you can download a list of blocked IP addresses into a log file or unblock IP addresses. 
+
+== Unblock Blocked IP Addresses
+
+. To unblock IP addresses, enter the IP addresses to unblock in the field, then click *Unblock IPs*. +
+. In the *Unblock an IP Address* field, enter an individual or a range of IP addresses. +
+For example, to unblock a range of IP addresses, enter: `192.0.2.0–192.0.2.255` +
```

Comment:
to unblock an ip address or range of IP addresses, enter an IP address and CIDR. i.e. To unblock a single IP address, enter: 192.0.2.0/32 or to unblock a range, enter: 192.0.2.0/24.